xroot/Overlay.lua

```diff
@@ -189,17 +237,18 @@ end
 
 local function onStartFloatingMenu()
```

Comment:
The current timer on this is 0.3 seconds, it might be nice to drop that down a bit to make it feel more snappy.

Reply:
I should make it configurable in the admin > settings.

Reply:
What value did you have in mind by the way?

Reply:
I only tried 0.1s and it felt pretty good, would probably want to try others as well.

Reply:
That was only in the emulator though, it may be different on device.

Reply:
Are the steps to compile a test-oriented firmware archive unclear?

Reply:
I have not yet been able to build a test firmware on OSX using docker. Granted I haven't tried _that_ hard to do it yet.

Last time I did there were some tricky build errors. I may give it another go.

Reply:
Ah.  I forgot you were on OSX using docker.  No problem.